File: bench/elixir/fib.exs
# Naive recursive Fibonacci — matches bench/fib.march (fib(40))
defmodule Fib do
  def fib(n) when n < 2, do: n
  def fib(n), do: fib(n - 1) + fib(n - 2)
end

IO.puts(Fib.fib(40))

File: bench/rust/fib.rs
// Naive recursive Fibonacci — matches bench/fib.march (fib(40))
fn fib(n: u64) -> u64 {
    if n < 2 { n } else { fib(n - 1) + fib(n - 2) }
}

fn main() {
    println!("{}", fib(40));
}
